Assets/Labrys/Generation/Grid.cs

```diff
@@ -134,7 +134,7 @@ public Vector2Int[] GetBoundary()
         /// </summary>
         /// <returns>The physical adjacencies.</returns>
         /// <param name="position">Position.</param>
-        public Connection GetPhysicalAdjacencies(Vector2Int position)
+        public Connection GetPhysicalAdjacencies_OLD(Vector2Int position)
```

Comment:
If this isn't being actively used, we should prune it.  Better to do it now while we're thinking about it.